src/types/dto/review.ts

```diff
@@ -58,3 +58,8 @@ export interface GithubCommentData {
   message: string
   createdAt: string
 }
+
+export interface BlobDiffDto {
+  old: string
+  new: string
```

Comment:
Hmm I think we should avoid using `new` as the key as it is reserved.

Reply:
yeap, thanks for pointing this out; slipped my mind and ts didn't show this to be an error @__@ will change to `oldValue/newValue`!